src/system/LinkExternal/LinkExternal.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Internal dependencies
+ */
+import { Link } from '../Link';
+import ScreenReaderText from '../ScreenReaderText';
+
+// Screen reader announcements
+const DEFAULT_EXTERNAL_LINK_TEXT = ', external link'; // reads as: link, <link text>, external link
+const NEW_TAB_TEXT = ', opens in a new tab'; // reads as: link, <link text>, external link, opens in a new tab
```

Comment:
I'm not sure why this wasn't translated in the dashboard since we have methods there from `i18n-calypso`. 

Do we not use `translate` from `i18n-calypso` in the design system to avoid having hard-coded text? Or would it make sense to implement and use here? If it's the former, we could implement the text in the dashboard using the `defaultScreenReaderText` and add translations there.

Reply:
Yeah it should have been translated in the VIP Dashboard. I don't think use translation in the Design System since we just pass the string through from the VIP Dashboard. We can also just start translation in the Design System if that is simpler.

Reply:
I don't mind adding i18n-calypso here.

Reply:
I added in: https://github.com/Automattic/vip-design-system/pull/390/commits/6491b233146c222c7528a01ec76e6d060b32e11e